src/Resources/config/services/services.xml

```diff
@@ -13,19 +13,19 @@
         </service>
 
         <service id="FINDOLOGIC\FinSearch\Findologic\Config\FinSearchSalesChannelExtension">
-            <tag name="shopware.entity.extension"/>
+            <tag name="shopware.entity.extension" />
         </service>
 
         <service id="FINDOLOGIC\FinSearch\Findologic\Config\FinSearchLanguageExtension">
-            <tag name="shopware.entity.extension"/>
+            <tag name="shopware.entity.extension" />
         </service>
 
         <service id="FINDOLOGIC\FinSearch\Findologic\Config\FindologicConfigService">
             <argument id="finsearch_config.repository" type="service" />
             <argument type="service" id="Doctrine\DBAL\Connection" />
         </service>
 
-        <service id="FINDOLOGIC\FinSearch\Struct\Config" class="FINDOLOGIC\FinSearch\Struct\Config" public="true">
+        <service id="FINDOLOGIC\FinSearch\Struct\Config" public="true">
```

Comment:
Removed all `class` attributes, as the `id` attribute with the class name is enough (And also done a lot of times already)